utils.py

```diff
@@ -456,18 +456,39 @@ def unescape_encoded_uri_component(escaped_string):
     return urllib.unquote(escaped_string).decode('utf-8')
 
 
-ASSET_DIR_PREFIX = None
+_CACHE_SLUG = None
+def get_cache_slug():
+    """Returns cache slug depending whether dev or prod."""
+    global _CACHE_SLUG # pylint: disable=global-statement
```

Comment:
Nope, it does not modify it many times. What do you mean by module-level variable? Isn't it already module level?